readthedocs/projects/models.py

```diff
@@ -753,7 +753,7 @@ class ProxitoURLConf:
 
         return ProxitoURLConf
 
-    @property
+    @cached_property
```

Comment:
Is this because it does a DB query? When will it be uncached? Should `superproject` also be a `cached_property`, and other methods here that hit the DB?

Reply:
yeah, +1 on more caching for these

Reply:
@stsewd you are right.